src/components/Single/SidePanel/Timeline.vue

```diff
@@ -1,56 +1,61 @@
 <template>
     <div id="container">
         <div class="search">
-            <SearchInput type="text" placeholder="Find wanderers"></SearchInput>
-        </div>
-        <div class="timeline">
-            <div class="item" data-id="1">
-                <HorizontalProfileView/>
-                <div class="content">
-                    We hebben een geweldige trip gehad naar het buitenland. <br>
-                    Zeker een aanrader.
-                </div>
-            </div>
-            <div class="item" data-id="2">
-                <HorizontalProfileView/>
-                <div class="content">
-                    We hebben een geweldige trip gehad naar het buitenland. <br>
-                    Zeker een aanrader.
-                </div>
-            </div>
-        </div>
-        <div class="createPost">
-            <!-- todo: handle item visibility on FAB click-->
-            <div class="FABItem">
-                <p>End trip</p>
-                <img src="img/ic_trip_end.svg"/>
-            </div>
-
-            <div class="FABItem">
-                <p>Text</p>
-                <img src="img/ic_text.svg"/>
-            </div>
-
-            <div class="FABItem">
-                <p>New trip</p>
-                <img src="img/ic_trip_start.svg"/>
-            </div>
-
-            <div class="FAB">
-                <img src="img/ic_add.svg"/>
-            </div>
```

Comment:
Why did this get removed?

Reply:
This definitely should not get removed

Reply:
<img width="85" alt="Screenshot 2019-12-09 at 16 00 55" src="https://user-images.githubusercontent.com/23239376/70446270-20298100-1a9d-11ea-9d82-244861f8015d.png">
it's this